ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+    },
+    {
+      id: 'Zadnor Saunion Magitek Halo',
+      netRegex: NetRegexes.startsUsing({ source: 'Saunion', id: '5DB5', capture: false }),
+      response: Responses.getUnder(),
+    },
+    {
+      id: 'Zadnor Saunion Magitek Crossray',
+      netRegex: NetRegexes.startsUsing({ source: 'Saunion', id: '5DB7', capture: false }),
+      alertText: (_data, _matches, outputs) => outputs.text(),
+      outputStrings: {
+        text: {
+          en: 'Go Intercardinals',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Saunion Mobile Halo',
+      netRegex: NetRegexes.startsUsing({ source: 'Saunion', id: ['5DB9', '5DBA', '5DBB', '5DBC'], capture: false }),
+      alertText: (_data, _matches, outputs) => outputs.text(),
+      outputStrings: {
+        text: {
+          en: 'Get Under (towards charge)',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Saunion Mobile Crossray',
+      netRegex: NetRegexes.startsUsing({ source: 'Saunion', id: ['5DBD', '5DBE', '5DBF', '5DC0'], capture: false }),
+      suppressSeconds: 5,
+      alertText: (_data, _matches, outputs) => outputs.text(),
+      outputStrings: {
+        text: {
+          en: 'Go Intercards (away from charge)',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Saunion Anti-Personnel Missile',
+      netRegex: NetRegexes.startsUsing({ source: 'Saunion', id: '5DC2' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Zadnor Saunion Missile Salvo',
+      netRegex: NetRegexes.startsUsing({ source: 'Saunion', id: '5DC3' }),
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Zadnor Saunion Wildfire Winds',
+      netRegex: NetRegexes.startsUsing({ source: 'Dawon The Younger', id: '5DCD', capture: false }),
+      delaySeconds: 10,
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          // TODO: during spiral scourge could be "get under middle/outer light orb"?
+          en: 'Get Under Light Orb',
+          de: 'Unter einem Lichtorb stellen',
+          fr: 'Allez sous un Orbe lumineux',
+          ja: '白玉へ',
+          cn: '靠近白球',
+          ko: '하얀 구슬 안으로',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Saunion Tooth and Talon',
+      netRegex: NetRegexes.startsUsing({ source: 'Dawon The Younger', id: '5DD4' }),
+      condition: tankBusterOnParty(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Zadnor Saunion Swooping Frenzy',
+      netRegex: NetRegexes.startsUsing({ source: 'Dawon The Younger', id: '5DD0', capture: false }),
+      infoText: (data, _matches, output) => {
+        // Every other Swooping Frenzy is followed by a Frigid Pulse, starting with the first.
+        data.saunionSwoopingCount = (data.saunionSwoopingCount || 0) + 1;
+        if (data.saunionSwoopingCount % 2)
+          return output.text();
+      },
+      outputStrings: {
+        text: {
+          en: 'Follow Dawon',
```

Comment:
Is there more than one boss?
```suggestion
          en: 'Follow Boss',
          de: 'Folge dem Boss',
          fr: 'Suivez le Boss',
          ja: 'ボスの後ろに追う',
          cn: '跟紧在Boss身后',
          ko: '보스 따라가기',
```

Reply:
Yeah, there's more than one boss, that's why it reads differently than Castrum.